meta-resin-common/recipes-core/initrdscripts/initramfs-framework_%.bbappend

```diff
@@ -1,14 +1,24 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 SRC_URI_append = " file://machineid \
+                   file://ro-rootfs \
                  "
 
 do_install_append() {
     install -m 0755 ${WORKDIR}/machineid ${D}/init.d/91-machineid
+
+    install -m 0755 ${WORKDIR}/ro-rootfs ${D}/init.d/89-ro-rootfs
 }
 
+BAD_RECOMMENDATIONS += "initramfs-module-rootfs"
```

Comment:
This is supposed to be used in images and I don't think you need it here anyway.